src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -38,9 +38,12 @@ def to_internal_value(self, data):
 
 class SchemaField(serializers.Field):
     def to_internal_value(self, data):
-        if not data:
+        if data is None:
```

Comment:
Would there be harm in `return {}` for this condition?

Reply:
I _think_ that we wouldn't want to do that because if someone wasn't updating the schema then we won't want to override what they already have. Like if it's `None` I thought that would be the case that they are updating other attributes through the API